gatsby/src/components/i18n/i18n.tsx

```diff
@@ -1,7 +1,7 @@
 import { useStaticQuery, graphql } from 'gatsby';
 import { useLocation } from '@reach/router';
 
-const gLang = (): string => {
+export const gLang = (): string => {
```

Comment:
To mě ani nenapadlo, že by se to hodilo exportovat, ale jo, je to super nápad. Zatím jsem to řešil různými hacky, bylo to na poslední chvíli, tak jsem nechtěl moc experimentovat, ale tohle mě fakt nenapadlo. 🤦‍♂️ 👍